### DISCUSSION TITLE
Will AspireUpdate talk to other repositories?

### DISCUSSION THREAD

**Discussion by asirota** [aspirepress/AspireUpdate#60] [Ideas]
There are other implementations being built to provide alternative update serversl to worg. Will AspireUpdate support other alternatives? Some sort of compatibility layer probably needs to be established to ensure the apis are relatively compatible to support this.

**Comment by namithj** [comment]
That will happen inside aspirecloud.  For the update plugin, the users selects one host from among the many mirrors and uses that.  Dealing with multiple hosts will be complicated at least initially as the dataset with different hosts might not be in sync leading to unwanted problems with updates.

**Reply by asirota** [reply]
> That will happen inside aspirecloud.

I don't think it will. There may be other comparable repo projects that share the api protocol agreed upon. You could choose a totally different end point. They don't even need to communicate with each other other than to be api compatible.

**Reply by namithj** [reply]
Thats what the API Host field is for.  We can add any compatible API host domain there

**Reply by asirota** [reply]
Agreed. But we would need to support Those other hosts assuming they do that. This discussion is to support that very effort of interoperability between repos.
